packages/schematics/angular/application/index.ts

```diff
@@ -307,6 +307,10 @@ export default function (options: ApplicationOptions): Rule {
     }
     validateProjectName(options.name);
     options.prefix = options.prefix || 'app';
+    // This line and the one above shouldn't be needed, but at the moment they are.
+    // This is because the default value defined in the schema.json file is not
+    // correctly set when this schematic is run from another schematic.
```

Comment:
We should probably open an issue and add a link to it here w/ a TODO to remove this when that issue is resolved.